django/home/models.py

```diff
@@ -940,6 +940,32 @@ def __str__(self):
         return "submission {} by {}".format(self.title, self.submitter)
 
 
+class DigestArchive(index.Indexed, models.Model):
```

Comment:
How about naming this `ComsesDigest` instead? An Archive is typically a collection of things, but an instance of this class represents a single issue of the comses digest.

Reply:
I guess technically it could be an ArchivedComsesDigest but overall I think ComsesDigest might work better. Happy to entertain other ideas though!